### DISCUSSION TITLE
Theme/Skin of Minimal Mistakes

### DISCUSSION THREAD

**Discussion by Ali-Morajabi** [academicpages/academicpages.github.io#2432] [Q&A]
Hi There! 

At First, I wanted to thank you for this template which eases things for us.
But now that I'm struggling with customizing my page, I can't find any section for changing the skin of it, which has been an 
option of the original Minimal Mistakes template and it was changeable in the `_config.yml` file and had several modes such as "aqua", "dark", "default", "air", "contrast" and so on.
I wanted to ask whether it is an option here in the template you've made or not!

I do appreciate any answer that could possibly help me here :)

**Comment by rjzupkoii** [comment]
Academic Pages was forked from Minimal Mistakes a couple years ago, so at this time it doesn't support changing the theme of the site. There are plans in the works to add that functionality with the 0.9 release, but right now the focus has been taking care of some of the technical debt that has accrued over time and adding some of the easier to achieve features.
